### PR TITLE
add reload model

### LINKED ISSUE #22: Add Model reload
Proposed enhancement: trigger model reload. Now it's impossible (?) to reload model. The workaround is to disable and reenable the node. It would be perfect to be able to automatize it.

### PR DESCRIPTION
**Pull request by acejacek**
Add trigger to reload the model.

### CODE REVIEW

**Comment by acejacek** [comment]
wrong commit :(

**Comment by bonastreyair** [comment]
Thanks for the PR! That's cool, if merged it will close #22

File: teachable_machine.html [hunk 1]
```diff
@@ -70,6 +70,8 @@ <h3>Inputs</h3>
         <dl class="message-properties">
             <dt>payload<span class="property-type">image buffer</span></dt>
                 <dd>A binary buffer of an image</dd>
+            <dt>reload<span class="property-type">string</span></dt>
```
Comment (bonastreyair) [inline teachable_machine.html:73]: any reason why not using a bool here?

File: teachable_machine.js [hunk 2]
```diff
@@ -190,6 +190,9 @@ module.exports = function (RED) {
 
     node.on('input', function (msg) {
       try {
+        if (node.ready && msg.reload !=='') {
```
Comment (bonastreyair) [inline teachable_machine.js:193]: missing a space: `if (node.ready && msg.reload !== '') {`

Comment (acejacek) [reply teachable_machine.js:193]: There are few problems with this commit. Please reject it all. I was too fast with it :(

Comment (bonastreyair) [reply teachable_machine.js:193]: close this PR then and start a fresh one if you like

**Pull request closed without merging**